### PR TITLE
fix(bin): skip first `ConsensusLayerHealthEvents` tick

### PR DESCRIPTION
**Pull request by shekhirin**
Skip the first tick that's ready immediately after `ConsensusLayerHealthEvents` initialization to prevent the false `ConsensusLayerHealthEvent::NeverSeen` event fired just after the node startup when it didn't have time to receive any CL updates.

```console
2023-06-06T08:28:17.003028Z  INFO reth::cli: reth 0.1.0 (64cbf522) starting
2023-06-06T08:28:17.003198Z  INFO reth::cli: Configuration loaded path="/home/ubuntu/.local/share/reth/sepolia/reth.toml"
2023-06-06T08:28:17.003215Z  INFO reth::cli: Opening database path="/home/ubuntu/.local/share/reth/sepolia/db"
2023-06-06T08:28:17.003878Z  INFO reth::cli: Database opened
2023-06-06T08:28:17.003889Z  INFO reth::cli: Starting metrics endpoint addr=127.0.0.1:9090
2023-06-06T08:28:17.004789Z  INFO reth::cli: Transaction pool initialized
2023-06-06T08:28:17.004805Z  INFO reth::cli: Connecting to P2P network
2023-06-06T08:28:17.004899Z  INFO net::peers: Loading saved peers file=/home/ubuntu/.local/share/reth/sepolia/known-peers.json
2023-06-06T08:28:17.036878Z  INFO reth::cli: Connected to P2P network peer_id=0xa4a8…c77c local_addr=0.0.0.0:30303
2023-06-06T08:28:17.037014Z  INFO reth::cli: Consensus engine initialized
2023-06-06T08:28:17.037028Z  INFO reth::cli: Engine API handler initialized
2023-06-06T08:28:17.037218Z  INFO reth::cli: RPC auth server started
2023-06-06T08:28:17.037287Z  INFO reth::cli: RPC IPC server started url=/tmp/reth.ipc
2023-06-06T08:28:17.037292Z  INFO reth::cli: RPC HTTP server started url=0.0.0.0:8545
2023-06-06T08:28:17.037297Z  INFO reth::cli: Starting consensus engine
2023-06-06T08:28:17.038210Z  INFO reth::cli: Status connected_peers=0 stage=None checkpoint=0
2023-06-06T08:28:17.038260Z  WARN reth::cli: Post-merge network, but never seen beacon client. Please launch one to follow the chain!
```

### CODE REVIEW

**Review by mattsse — CHANGES_REQUESTED** [review]
I'm skeptical that a new function that waits for 300s is the right fix here when we could track this in the stream impl

File: bin/reth/src/node/mod.rs [hunk 1]
```diff
@@ -355,6 +355,7 @@ impl Command {
             if self.debug.tip.is_none() {
                 Either::Left(
                     ConsensusLayerHealthEvents::new(Box::new(blockchain_db.clone()))
+                        .await
```
[the remark this answers is not in the document]
Comment (shekhirin) [reply bin/reth/src/node/mod.rs:358]: it doesn't iiuc?

> The first tick completes immediately.

from https://docs.rs/tokio/latest/tokio/time/fn.interval.html

[the remark this answers is not in the document]
Comment (mattsse) [reply bin/reth/src/node/mod.rs:358]: I yes you're right,
however we can use https://docs.rs/tokio/latest/tokio/time/fn.interval_at.html `now + interal` to get the same effect?

[the remark this answers is not in the document]
Comment (shekhirin) [reply bin/reth/src/node/mod.rs]: ooo right, that's much better. fixed.

**Review by mattsse — APPROVED** [review]

**Pull request merged**